guides/07_FAQs.md

```diff
@@ -145,6 +145,17 @@ The possible reasons are as follows:
 2. The close code `4410` from the Chime backend indicates that an attendee has attempted to join an already-ended meeting. The Chime SDK for JavaScript throws an error with the status code `MeetingEnded`.
 3. The close code between `4500` and `4599` (inclusive) indicates an internal server error in the Amazon Chime backend. In this case, the Chime SDK for JavaScript throws an error with the status code `SignalingInternalServerError`. Please create a GitHub issue including the Chime SDK browser logs.
 
+
+### What is the client behavior when WebSocket connection is terminated during connection?
+
+If the WebSocket connection is terminated when clients are in the [JoinAndReceiveIndexTask](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/task/JoinAndReceiveIndexTask.ts) or [SubscribeAndReceiveSubscribeAckTask](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/task/SubscribeAndReceiveSubscribeAckTask.ts), they will get `TaskFailed`, `InternalServerError` or `SignalingBadRequest` meeting session status code depending on failure reasons. If the status code is `TaskFailed` or `InternalServerError`, clients will reconnect immediately without waiting for connection timeout. If the status code is `SignalingBadRequest`, clients will not reconnect. There is a customizable [reconnectTimeout](https://github.com/aws/amazon-chime-sdk-js/blob/0147ed9fb76429a70fb0161405687af63b592fe0/src/meetingsession/MeetingSessionConfiguration.ts#L67) for builder to decide how long they want to timeout the reconnection. The default value is 2 minutes.
+
+### What happens if the client is in mid call and their WebSocket connection is terminated?
```

Comment:
Request you to check above comment for "client". Should we club the explanation of both of these questions together as they have related foundation? The MonitorTask can be explained below other tasks and then explain as you do in this answer.